src/goldsky/evm/evm-mapping.ts

```
@@ -1,49 +1,75 @@
-import { BigInt, Bytes, ethereum } from '@graphprotocol/graph-ts';
-import {
-  Deposit,
-  Withdrawal,
-  PriceUpdate
-} from '../generated/VaultContract/VaultContract';
-import { Vault, Transaction, PriceUpdate as PriceUpdateEntity } from '../generated/schema';
+import { BigInt, Bytes, ethereum, Address } from '@graphprotocol/graph-ts';
```

Biome (1.9.4)

[error] Do not shadow the global "BigInt" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)